src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -7,32 +7,33 @@
 using System.Linq;
 using System.Net;
 using System.Net.Http;
-using System.Text;
+using System.Net.Http.Headers;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
-using Octokit;
 
 namespace Microsoft.Docs.Build
 {
-    internal class GitHubAccessor
+    internal sealed class GitHubAccessor
     {
         private static readonly HttpClient s_httpClient = new HttpClient();
```

Comment:
As the constructor will add token to this client, suggest not to use static.

Reply:
good suggestion, but need change a lot of class to IDispoase...